test.sh

```diff
@@ -635,8 +635,21 @@ function dep_pass {
 
 function release_pass {
   rm -f ./bin/etcd-last-release
-  # to grab latest patch release; bump this up for every minor release
-  UPGRADE_VER=$(git tag -l --sort=-version:refname "v3.4.*" | head -1)
```

Comment:
For the stable release (3.5 in this case), hard code isn't a problem. Its previous version must be 3.4.x.

Reply:
Hmm I do think it will be beneficial if our 3.5 release tests will run against the latest patch of `3.4.x`.  That way we get assurance our backports to 3.4 have not introduced an issue with the upgrade process.